src/mokka/src/test/java/pl/hycom/mokka/emulator/logs/LogsControllerTest.java

```diff
@@ -0,0 +1,106 @@
+package pl.hycom.mokka.emulator.logs;
+
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.security.test.web.servlet.setup.SecurityMockMvcConfigurers;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import pl.hycom.mokka.AbstractTest;
+import pl.hycom.mokka.emulator.logs.model.Log;
+
+import java.net.URI;
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * @author Kamil Adamiec (kamil.adamiec@hycom.pl)
+ */
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class LogsControllerTest extends AbstractTest {
```

Comment:
As it is an integration tests and it takes much more time to execute than regular unit tests I would be in favor to introduce naming convention that in future may allow us to execute those tests in different build phases.
LogsControllerTest -> LogsControllerIntegrationTest